input/fsh/expansion.fsh

```diff
@@ -0,0 +1,8 @@
+Instance: expansion
```

Comment:
Is this Instance name explain "what is it"?

Reply:
This is not an example, rather it is a configuration resource to say that the FHIR validator should use Swedish SNOMED CT when validating: https://fshschool.org/docs/sushi/tips/#choosing-a-snomed-ct-edition